### PR TITLE
Feature/cli refactor

### PR DESCRIPTION
**Pull request by rjullman**
Create one cli that orchestrates all other clis.

### CODE REVIEW

**Comment by rjullman** [comment]
I called the cli "titan", but maybe it should be renamed something less punny.

**Comment by rjullman** [comment]
I also removed the default config path since this now all the clis are combined and we probably either want the config path always required or a hardcoded default (that we can decide later).

File: atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/impl/AtlasDbServicesImpl.java [hunk 1]
```diff
@@ -39,18 +42,51 @@
...
+                config.getConfig(),
+                Optional.<SSLSocketFactory>absent(),
+                ImmutableSet.<Schema>of(),
                 new TransactionManagers.Environment() {
                     @Override
                     public void register(Object resource) {
                     }
-                }, false);
+                },
+                true);
```
Comment (clockfort) [inline atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/impl/AtlasDbServicesImpl.java]: was there a reason before to allowHiddenTableAccess = false here?

Comment (rjullman) [inline atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/impl/AtlasDbServicesImpl.java]: I think I just wasn't paying attention.  I think we might as well allow the clis to mess with hidden tables?  Any objection?

Comment (clockfort) [inline atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/impl/AtlasDbServicesImpl.java]: No objection, your honor.

**Pull request merged**